less/dropdowns.less

```diff
@@ -212,6 +212,20 @@
   right: auto;
 }
 
+@media (max-width: @screen-sm-min) {
+  .dropdown-menu-bottom {
```

Comment:
I'm finding this naming slightly confusing because we have `dropdown-menu-left` and `dropdown-menu-right`, and they do quite different things.  How about `dropdown-menu-native`?